src/dsymbol/conversion/first.d

```diff
@@ -1341,7 +1342,42 @@ class InitializerVisitor : ASTVisitor
 
 	// Skip these
 	override void visit(const ArgumentList) {}
-	override void visit(const NewAnonClassExpression) {}
+
+	override void visit(const NewExpression ne)
+	{
+		if (ne.newAnonClassExpression)
+		{
+			(cast() ne).assignExpression = lowerNewAnonToNew(ne);
```

Comment:
This must be in the NewExpression parameters, no as its assignExpression which is used for totaly different things. The DCD PR works for some reasons...